charts/kamaji-etcd/templates/etcd_headless_service.yaml

```diff
@@ -12,7 +12,5 @@ spec:
       name: client
     - port: {{ .Values.peerApiPort }}
       name: peer
-    - port: {{ .Values.metricsPort }}
```

Comment:
Why are we removing the metrics port?

Maybe because it's useless since we use PodMonitor?

Reply:
Made metrics port label selector explicit.